domains/just.json

```
@@ -5,6 +5,12 @@
     "discord": "1117482901353812088"
   },
   "records": {
-    "A": ["185.199.108.153", "185.199.109.153", "185.199.110.153", "185.199.111.153"]
-  }
+    "URL": "https://just.js.org/"
+  },
+  "redirect_config": {
+    "custom_paths": {
+      "/docs": "https://just.js.org/docs",
+      "/code": "https://just.js.org/code"
+    },
+    "redirect_paths": true
```

you dont need to add thus since /docs and /code are in the same base URL
```suggestion
  }
```

---

ok ty